### DISCUSSION TITLE
Developer Update - September 2024

### DISCUSSION THREAD

**Discussion by w3b6x9** [supabase/supabase#29828] [Changelog]
We’ve announced a Vercel partnership, we’re hosting an AI hackathon with our friends at Y Combinator, and we raised $80M. Let’s dive right in:

## Supabase + Vercel Partnership

![Supabase + Vercel Partnership](https://obuldanrptloktxcffvn.supabase.co/storage/v1/object/public/images/marketing-emails/september2024/supabase-x-vercel.png?t=2024-10-10T02%3A59%3A39.452Z)

We released an official Vercel integration. You can quickly spin up Supabase projects from Vercel’s dashboard, with full support for Vercel templates like Supabase Starter and integrated billing through Vercel.

[Full announcement](https://supabase.link/supabase-vercel-partnership)

## Revamped Users Management UI in Studio

![Revamped Users Management UI in Studio](https://obuldanrptloktxcffvn.supabase.co/storage/v1/object/public/images/marketing-emails/september2024/revamped-users-management-ui-in-studio.png?t=2024-10-10T03%3A00%3A01.836Z)

Our Frontend team revamped the Auth section in Studio. You now have access to more user details, ban-user functionality, authenticated logs grouped by user, sort columns based on your preference, and a few other features requested by the community.

[Changelog](https://supabase.link/users-mgmt-revamp)

## Edge Functions are 2x smaller and boot 3x faster

![Edge Functions are 2x smaller and boot 3x faster](https://obuldanrptloktxcffvn.supabase.co/storage/v1/object/public/images/marketing-emails/september2024/edge-functions-are-2x-smaller-and-boot-3x-faster.png?t=2024-10-10T03%3A00%3A13.326Z)

The Edge Functions team has reduced function sizes by half and boot times by 300% in most cases when importing npm modules by lazy evaluating dependencies and reducing package section sizes as well as switching to the xxHash-3 hash algorithm.

[Blog post](https://supabase.link/edge-functions-faster-smaller)

## Supabase AI Hackathon at Y Combinator

![Supabase AI Hackathon at Y Combinator](https://obuldanrptloktxcffvn.supabase.co/storage/v1/object/public/images/marketing-emails/september2024/supabase-ai-hackathon-at-y-combinator.png?t=2024-10-10T03%3A00%3A23.124Z)

On November 22 + 23 we’re hosting an AI-focused hackathon at Y Combinator in San Francisco. We’re welcoming anyone to apply and build the next wave of AI applications. The panel of judges will include our founders, [Paul Copplestone](https://x.com/kiwicopple) and [Ant Wilson](https://x.com/AntWilson), and YC partners.

[Full announcement](https://supabase.link/supabase-yc-ai-hackathon)

## Launch Week 12 Hackathon Winners

![Launch Week 12 Hackathon Winners](https://obuldanrptloktxcffvn.supabase.co/storage/v1/object/public/images/marketing-emails/september2024/launch-week-12-hackathon-winners.png?t=2024-10-10T03%3A00%3A42.904Z)

Many high quality projects were submitted for LW12’s hackathon but our panel of judges selected [Whisker Jam](https://supabase.link/whisker-jam-lw12) as Best Overall Project because of its cuteness (who doesn’t love cats), funky musical instruments, and multiplayer functionality. Congratulations 👏 to [@n0t_buddy](https://twitter.com/n0t_buddy) who will receive the prize of mechanical keyboards.

[Full list of winners](https://supabase.link/lw12-hackathon-winners) | [All the submissions](https://supabase.link/lw12-submissions)

## Quick Product Announcements

<ul>
    <li>
      [Dashboard] Schema Visualizer nodes are now persisted [<a href="https://supabase.link/schema-visualizer-nodes">Changelog</a>]
    </li>
    <li>
      [Edge Functions] Serverless image transformations with ImageMagick (via Wasm) [<a href="https://supabase.link/imagemagick-wasm">Docs</a>]
    </li>
    <li>
      [Infra] Projects on compute instances XL and larger can create up to 5 Read Replicas  [<a href="https://supabase.link/5-read-replicas">Changelog</a>]
    </li>
    <li>
      [Storage] XHTML responses only work with a Custom Domain [<a href="https://supabase.link/xhtml-custom-domains">Changelog</a>]
    </li>
    <li>
      [Billing] Paid projects have an upgrade from Nano to Micro instance at no additional cost [<a href="https://supabase.link/micro-upgrade">Docs</a>]
    </li>
  </ul>

## Community Highlights

![community highlights](https://obuldanrptloktxcffvn.supabase.co/storage/v1/object/public/images/marketing-emails/september2024/community_highlights.png?t=2024-10-10T03%3A00%3A54.371Z)

*   Using Cursor to have AI build out a social network app powered by Next.js and Supabase \[[Video](https://supabase.link/cursor-ai-app-nextjs-supabase)\]
*   Wordle Teams. Compete with friends, keep score to establish bragging rights in the ultimate app for Wordle enthusiasts \[[Repo](https://supabase.link/wordle-teams)\]
*   Next.js + TanStack Query + Supabase + Supabase Cache Helpers: a detailed tutorial on how to implement this stack in your application \[[Article](https://supabase.link/nextjs-tanstack-query-supabase-guide)\]
*   Supabase Auth: The Ultimate Authentication Solution for Cross-Platform Apps using React Native \[[Article](https://supabase.link/supabase-auth-ultimate-solution-react-native)\]
*   How to build local-first Expo Apps \[[Video](https://supabase.link/local-first-expo-supabase)\]

## Supabase $80 Million Series C

We raised $80 million Series C in an up round that brings our total funding to $196 million. This round was led by Peak XV and Craft Ventures with participation by Avra Capital and previous investors Coatue, Felicis, and Y Combinator.

[Full announcement](https://supabase.link/supabase-80-mil-series-c)

<hr /><em>This discussion was created from the release <a href='https://github.com/supabase/supabase/releases/tag/v1.24.09'>Developer Update - September 2024</a>.</em>

**Comment by KnudH** [comment]
You missed the fact, that you also released some changes to self-hosting and added supavsior: https://github.com/supabase/supabase/pull/29596

In my case, this breaks the complete instance because the new database _supabase does not get created automatically.
